client/src/views/admin/AdminList.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <b-container>
+    <table class="table">
+      <thead>
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">Name</th>
+          <th scope="col">Role</th>
+          <th scope="col">Username</th>
+          <th scope="col">Email</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(adminDetails, index) in admins" :key="index">
+          <td>{{ index + 1 }}</td>
+          <td>{{ adminDetails.name }}</td>
+          <td>{{ adminDetails.role }}</td>
+          <td>{{ adminDetails.userName }}</td>
+          <td>{{ adminDetails.email }}</td>
```

Comment:
you can also add option for admin to remove,disable, verify a user. 
let me know if you feel any issues, or I can add this feature maybe..?

Reply:
Hey, Currently we have API's for remove. Updated the screens for remove. Will add other once verify and disable API's are completed.